dbms/src/Storages/StorageTinyLog.cpp

```diff
@@ -428,6 +428,12 @@ void StorageTinyLog::truncate(const ASTPtr &, const Context &, TableStructureWri
         addFiles(column.name, *column.type);
 }
 
+void StorageTinyLog::drop(TableStructureWriteLockHolder &)
+{
+    std::unique_lock<std::shared_mutex> lock(rwlock);
+    disk->clearDirectory(table_path);
```

Comment:
Shouldn't `table_path` itself be deleted as well?

Reply:
Then we need to add `void remove(bool recursive);` in IDisk interface.